ansible/roles/ocp4-workload-quay-operator/tasks/workload.yml

```diff
@@ -30,9 +30,12 @@
 
 - name: Generate Quay Superuser Password
   when:
-  - ocp4_workload_quay_operator.superuser_password is not defined or ocp4_workload_quay_operator.superuser_password|length == 0
+  - >-
```

Comment:
This reads strange using the folded style block. How about something like:

```
when: ocp4_workload_quay_operator.superuser_password is not defined or
      ocp4_workload_quay_operator.superuser_password | length == 0
```

Reply:
Removed the list notation `-`. But had to preserve the `>-` operator due to line length

Reply:
Ok, I'll leave this since it looks better. Remember though, if you have a workload that just makes it hard to write these due to line length it is okay to change the yamllint file to be longer or just disable it. We don't want to use the letter of the rule to defeat the spirit of the intent.